core/shared/src/main/scala/caseapp/core/help/CommandHelp.scala

```diff
@@ -17,6 +19,7 @@ import dataclass.data
   def helpMessage(progName: String, commandName: Seq[String]): String = {
     val b = new StringBuilder
     b ++= s"Command: ${commandName.mkString(" ")}${Help.NL}"
+    b ++= helpMessage.map(_.message).getOrElse("")
```

Comment:
NIT: only add this one to `b` if `helpMessage` is non empty? So that we don't get an empty line if there's no help message.

```suggestion
    for (m <- helpMessage)
      b ++= m.message
```

Reply:
Ah yes totally i did it for the other case but somehow didn't do it here 😅